### PR TITLE
fix: nodejs 6 async_hook issue

### PR DESCRIPTION
**Pull request by fabiogomessilva**
Fix for the issue found on PR/6

### CODE REVIEW

File: packages/opencensus-core/src/trace/model/tracer.ts [hunk 1]
```diff
@@ -52,7 +52,6 @@ export class Tracer implements types.Tracer {
   constructor() {
     this.activeLocal = false;
     this.contextManager = cls.createNamespace();
-    this.clearCurrentTrace();
```
Comment (kjin) [inline packages/opencensus-core/src/trace/model/tracer.ts]: What is the purpose of this deletion?

File: packages/opencensus-core/test/test-tracer.ts [hunk 2]
```diff
@@ -52,17 +52,13 @@ describe('Tracer', () => {
 
   /** Should get/set the current RootSpan from tracer instance */
   describe('get/set currentRootSpan()', () => {
-    let tracer, rootSpan;
-    before(() => {
-      tracer = new Tracer();
-      rootSpan = new RootSpan(tracer);
-      tracer.currentRootSpan = rootSpan;
-    });
+    const tracer = new Tracer().start(defaultConfig);
```
Comment (kjin) [inline packages/opencensus-core/test/test-tracer.ts]: This change looks unrelated to the Node 6 change. Was this intended?

File: packages/opencensus-core/src/trace/model/tracer.ts (hunk 1 above)
Comment (fabiogomessilva) [reply packages/opencensus-core/src/trace/model/tracer.ts]: Some unit tests were failing. Both cases related to context issues.

File: packages/opencensus-core/test/test-tracer.ts (hunk 2 above)
Comment (fabiogomessilva) [reply packages/opencensus-core/test/test-tracer.ts]: Some unit tests were failing. Related to context issues.

File: packages/opencensus-core/src/trace/model/tracer.ts (hunk 1 above)
Comment (kjin) [reply packages/opencensus-core/src/trace/model/tracer.ts]: `clearCurrentTrace` is part of the public API but it will throw when using `continuation-local-storage`. Could you add a TODO to address this in a follow-up PR?

**Comment by kjin** [comment]
Current CircleCI failure is unrelated (Circle seems to default to Node 4 for some reason.)
